spec/support/authorization_stubs.rb

```diff
@@ -23,4 +23,11 @@ def allow_operations(operation, operation_args)
 
     allow(AUTHORIZER_CLASS).to receive(:new).with(Hash).and_return(authorizer)
   end
+
+  def expect_operation(operation, *args, authorizer: instance_double(AUTHORIZER_CLASS))
+    expect(authorizer).to receive(operation).with(*args)
+
+    allow(AUTHORIZER_CLASS).to receive(:new).with(Hash).and_return(authorizer)
+    authorizer
+  end
```

Comment:
We hopefully don't need to add this new method here, if we can satisfy the spec in https://github.com/venuu/jsonapi-authorization/pull/52/files#r109100787 with the existing matchers